File: 2/z0.m
a = [1:100];
b = [1:2:100];
c = [-pi:0.01*pi: pi];
d = [[-pi: 0.01*pi: -0.01*pi] [0.01*pi : 0.01*pi : pi] ];
e = (sin(a) > 0) .* sin(a);

A = transpose(reshape([1:100],10,10));
B = diag([1:100]) + diag([1:99], -1) + diag([1:99], 1);
C = triu(ones(10,10));
% a = [1:5]
D = [arrayfun(@(x) (x * (x + 1)) / 2.0, a);
     factorial(a)];
E1 = repmat([1:100], 100, 1);
E2 = repmat(reshape([1:100],100, 1), 1, 100);
E = mod(E2,E1) == 0;

File: 1/z0.py
import numpy as np
from numpy import pi as π
a = np.arange(1, 101)
b = np.arange(1, 100, 2)
c = np.linspace(-π, π, 201)
d = np.concatenate((np.linspace(-π, -π / 100, 100), np.linspace(π / 100, π, 100)))
e = np.clip(np.sin(a), 0, None)
A = np.arange(1, 101).reshape(10, 10) 
B = (
    np.diagflat(np.arange(1,101)) +
    np.diagflat(np.arange(1,100), k=-1) + 
    np.diagflat(np.arange(1,100), k=1)
    )
C = np.triu(np.ones(10))
D3 = np.concatenate(
        (
        np.cumsum(np.arange(1, 101).astype('float')).reshape(1,100),
        np.cumprod(np.arange(1,101).astype('float')).reshape(1,100)
        ), 
    axis=0)
E = (
        (
            (np.arange(1,101).reshape(100,1) @ np.ones((1, 100), dtype='int')) %
                (np.ones((100, 1), dtype='int') @ np.arange(1, 101).reshape(1, 100)) 
        ) == 0
    ).astype('int')

# print(a,b,c,d,e)
# print(A,B,C,D,E)
